dockerfiles/cli/tests/cmd_config_tests.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+# Copyright (c) 2017 Codenvy, S.A.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Roman Iuvshyn
+
+load '/bats-support/load.bash'
+load '/bats-assert/load.bash'
+source /dockerfiles/cli/tests/test_base.sh
+
+teardown() {
+  docker run --rm -v "${SCRIPTS_DIR}":/scripts/base -v /var/run/docker.sock:/var/run/docker.sock -v "${tmp_path}":/data $CLI_IMAGE destroy --quiet --skip:nightly --skip:pull
+}
+
+@test "test CLI 'config' command" {
+  #GIVEN
+  tmp_path="${TESTRUN_DIR}"/cli_cmd_config
+  container_tmp_path="${CONTAINER_TESTRUN_DIR}"/cli_cmd_config
+
+  #WHEN
+  docker run --rm -v "${SCRIPTS_DIR}":/scripts/base -v /var/run/docker.sock:/var/run/docker.sock -v "${tmp_path}":/data $CLI_IMAGE config --skip:nightly --skip:pull
+
+  #THEN
+  [[ -d "${container_tmp_path}"/docs ]]
+  [[ -e "${container_tmp_path}"/che.env ]]
+  [[ -e "${container_tmp_path}"/cli.log ]]
+  [[ -d "${container_tmp_path}"/instance ]]
+  [[ -d "${container_tmp_path}"/instance/config ]]
+  [[ -f "${container_tmp_path}"/instance/config/che.env ]]
+  [[ -d "${container_tmp_path}"/instance/data ]]
+  [[ -d "${container_tmp_path}"/instance/logs ]]
+  [[ -d "${container_tmp_path}"/instance/stacks ]]
+  [[ -d "${container_tmp_path}"/instance/templates ]]
+  [[ -f "${container_tmp_path}"/instance/docker-compose-container.yml ]]
+  [[ -f "${container_tmp_path}"/instance/che.ver.do_not_modify ]]
```

Comment:
Yeah, it should be fine, I was just wondering why it is mixed, whether some of those files are just symlinks or whatever :)